pandas/sparse/frame.py

```diff
@@ -77,29 +79,26 @@ def __init__(self, data=None, index=None, columns=None, default_kind=None,
             data = {columns[0]: data}
 
         if default_fill_value is None:
-            default_fill_value = np.nan
+            default_fill_value = 0 if is_scipy_sparse(data) else np.nan
```

Comment:
Well, SciPy sparse structures imply missing values are zeros. It felt like a sane default, but really it's better to keep the default _nan_. At least that way the user can decide, what fill values they want. (And I don't know how to "unfill" a sparse frame that has been `fillna`-d.)